### PR TITLE
Release note -  alert improvements

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**

### CODE REVIEW

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ To see the current status of the Mendix Developer Portal and Control Center, see
 
 ## 2022
 
+### November 10th, 2022
+
+#### Improvements
+
+* We updated the `Virtual Machine Out of Memory` and `Virtual Machine Error` alerts to offer more clarity and visibility into the application behavior. These changes are available after you redeploy your app.
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/developer-portal/_index.md]: Code formatting the best option here? 

Also don't forget to update https://docs.mendix.com/releasenotes/developer-portal/deployment/

and the date on the home page for the release.

**Comment by katarzyna-koltun-mx** [comment]
Probably won't make the 10/11 buildpack, waiting for the go ahead from @denis-shuvalov

File: content/en/docs/releasenotes/developer-portal/_index.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/releasenotes/developer-portal/_index.md]: Changed code formatting to italics.

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/developer-portal/_index.md]: Maybe these should be bold, do they appear in the UI @katarzyna-koltun-mx ?

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/developer-portal/_index.md]: Also:

>Also don't forget to update https://docs.mendix.com/releasenotes/developer-portal/deployment/

and the date on the home page for the release.

Comment (katarzyna-koltun-mx) [reply content/en/docs/releasenotes/developer-portal/_index.md]: I think someone else has updated both dates already. For the alert names, I don't know if they appear in the UI or just in email, but italics are used for them elsewhere in the document.

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/developer-portal/_index.md]: Which document? 

Is this for Control Center, or for a cloud facility? Is this actually the correct place for this note?

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/developer-portal/_index.md]: I think these should be bolded, if they're referring to error messages in the UI: https://docs.mendix.com/developerportal/operate/monitoring-application-health/

Also, this seems to be cloud, not just Dev Portal or Control Center, so probably needs to be published here?: https://docs.mendix.com/releasenotes/developer-portal/mendix-cloud/

Comment (katarzyna-koltun-mx) [reply content/en/docs/releasenotes/developer-portal/_index.md]: Done.

**Pull request merged**